netci.groovy

```diff
@@ -104,6 +104,15 @@ def static isCorefxTesting(def scenario) {
     return scenario.substring(0,corefx_prefix.length()) == corefx_prefix
 }
 
+def static isArm64Testing(def newJob) {
```

Comment:
Also, are there any ways to make sure that the timeout is set correctly for these jobs that were generated in the test CI run?